data-prepper-core/src/main/java/org/opensearch/dataprepper/plugin/DefaultPluginFactory.java

```diff
@@ -91,6 +91,7 @@ private <T> PluginArgumentsContext getConstructionContext(final PluginSetting pl
 
         return new PluginArgumentsContext.Builder()
                 .withPluginSetting(pluginSetting)
+                .withPipelineDescription(pluginSetting)
```

Comment:
Can you update the unit test for this class to verify this?

Reply:
Will add unit test.